### DISCUSSION TITLE
V2.2.6 log and config file

### DISCUSSION THREAD

**Discussion by s9rt1v** [milvus-io/milvus#31467] [Q&A and General discussion]
I have deployed milvus 2.2.6 serve on rancher by using docker, but I'm confronted with occassional crash during continuously seach in python, where can I find log and config file to locate the problem

**Comment by yhmo** [comment]
There should be a container named "milvus-standalone". 
Use **docker ps** command to show all the containers and their id. 
Use **docker logs xxxxx** to show the log of a container, the "xxxxx" is the id of the milvus-standalone container.

Milvus configurations file is inside the container, use **docker exec -it xxxxx bash** to enter the container, you will see the milvus.yaml under the path **/milvus/configs**
